pkg/commands/copy.go

```diff
@@ -59,11 +59,15 @@ func (c *CopyCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.Bu
 		if err != nil {
 			return err
 		}
+		if fi.IsDir() && !strings.HasSuffix(fullPath, string(os.PathSeparator)) {
+			fullPath += "/"
+		}
 		cwd := config.WorkingDir
 		if cwd == "" {
 			cwd = constants.RootDir
 		}
-		destPath, err := util.DestinationFilepath(src, dest, cwd)
+
+		destPath, err := util.DestinationFilepath(fullPath, dest, cwd)
```

Comment:
Why `fullPath` instead of `src`?

Reply:
To check if source is a directory. Would it be right to call `IsDestDir(src)` once again inside `util.DestinationFilePath()`?